main.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from dotenv import load_dotenv
+
+from functions.instrument_functions import get_installed_instrument_data, \
```

Comment:
Could use - 

`from functions.instrument_functions import (get_installed_instrument_data, get_instruments_with_active_survey_day_today_and_cases ...) `

I think it looks better than backslash escapes `\` - up to you though :)

Reply:
Added a comment that local env vars aren't necessary for unit and behave tests.

Reply:
I agree it looks better, done.